qtpy/QtWebEngineWidgets.py

```diff
@@ -27,6 +27,18 @@
         from PyQt5.QtWebKitWidgets import QWebView as QWebEngineView
         from PyQt5.QtWebKit import QWebSettings as QWebEngineSettings
         WEBENGINE = False
+elif PYSIDE2:
+    try:
+        from PySide2.QtWebEngineWidgets import QWebEnginePage
+        from PySide2.QtWebEngineWidgets import QWebEngineView
+        from PySide2.QtWebEngineWidgets import QWebEngineSettings
+    except ImportError:
+        from PySide2.QtWebKitWidgets import QWebPage as QWebEnginePage
+        from PySide2.QtWebKitWidgets import QWebView as QWebEngineView
+        #: Current PySide2 builds seem to be missing this.
+        #: I'll leave it in for now because the final builds should have it
+        from PySide2.QtWebKit import QWebSettings as QWebEngineSettings
```

Comment:
It means that right now importing `QtWebEngineWidgets` will fail with pyside2 even if `QWebEngineSettings` is not used ? I'm in favor of commenting this line for now and readding it when pyside2 includes `QWebSettings`.